### DISCUSSION TITLE
Candidates for awesome-orgs

### DISCUSSION THREAD

**Discussion by eric-hc** [beansource/awesome-orgs#9] [Ideas]
Request and discuss awesome organizations ✨ 

🚀 : send it and add the org

**Comment by eric-hc** [comment]
[Svelte](https://github.com/sveltejs) also would be an obvious choice I think
- Verified
- Already big growing community
- Website, branding, etc.

**Reply by ghost** [reply]
kinda mid rn

**Comment by eric-hc** [comment]
Could also create a programming language section and start with [Dart](https://github.com/dart-lang) and [Go](https://github.com/golang)

**Reply by eric-hc** [reply]
[Rust](https://github.com/rust-lang)

**Reply by ghost** [reply]
meh on these

**Comment by eric-hc** [comment]
GraphQL section

**Reply by eric-hc** [reply]
https://github.com/apollographql

**Reply by eric-hc** [reply]
https://github.com/graphql

**Reply by eric-hc** [reply]
[Hasura](https://github.com/hasura)

**Comment by eric-hc** [comment]
Company candidates

**Reply by eric-hc** [reply]
[Cal](https://github.com/calcom)

**Comment by eric-hc** [comment]
[freecodecamp](https://github.com/freeCodeCamp) could introduce a learning section 🤔

**Comment by eric-hc** [comment]
[Remix](https://github.com/remix-run)

**Comment by eric-hc** [comment]
New developer tools section?

**Reply by eric-hc** [reply]
[Fig](https://github.com/withfig)

**Reply by eric-hc** [reply]
[OpenSauced](https://github.com/open-sauced), but more of a community thing

**Reply by eric-hc** [reply]
[Raycast](https://github.com/raycast)

**Reply by eric-hc** [reply]
REST tools 
- [HTTPie](https://github.com/httpie) 🥧
- [Postman](https://github.com/postmanlabs) Labs
- [Hoppscotch](https://github.com/hoppscotch) 👽

**Comment by eric-hc** [comment]
AI section with the inaugural https://github.com/openai

**Comment by ghost** [comment]
### Analytics section starting with

https://github.com/posthog
